web/src/components/Plugin/UI/referer-restriction.tsx

```diff
@@ -86,7 +90,7 @@ const RefererRestriction: React.FC<Props> = ({ form }) => {
                       </Form.Item>
                     </Col>
                     <Col style={{ ...removeBtnStyle, marginLeft: -10 }}>
-                      {fields.length > 1 ? (
+                      {fields.length > whiteMinLen ? (
```

Comment:
```suggestion
                      {fields.length > allowListMinLength ? (
```

Reply:
Thanks for your suggestion, I have modified it.